fastlane.gemspec

```diff
@@ -57,4 +57,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'webmock', '~> 1.19.0'
   spec.add_development_dependency 'coveralls'
   spec.add_development_dependency 'rubocop', '~> 0.29'
+  spec.add_development_dependency 'appium_lib', '~> 4.1.0'
```

Comment:
I tried to remove it from dependency. but spec failed because I wrote `require 'appium_lib'`
https://github.com/fastlane/fastlane/pull/935/files#diff-d3e4f0808df6e05aab7fce9a3a6cf589R14

Do you have any idea to fix this?